### PR TITLE
✅ reverse String passed test

### PR DESCRIPTION
**Pull request by alimahmoudidev**
I was able to add a function using return functions according to the string receiving structure
Of course, I know that this method may not be optimal due to the limitation of the sent parameter

### CODE REVIEW

File: src/04-reverseString/main.js [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+  }
+  return reverseStr;
 }
 
 function reverseStringV2(str) {
   // Using arrays:
+  return str.split("").reverse().join("");
+}
+
+function reverseStringV3(str) {
```
Comment (miladsade96) [inline src/04-reverseString/main.js:15]: We will talk about recursive approach in the upcoming session

Comment (alimahmoudidev) [reply src/04-reverseString/main.js:15]: Grateful
I look forward to the next session

Comment (miladsade96) [reply src/04-reverseString/main.js:15]: Thanks

**Comment by miladsade96** [comment]
Well done dear Ali,

I just pushed the functions implementations directly after our session is ended, because we needed to share updated file on the telegram channel. Feel free to rebase your pull request to resolve the conflicts and then i will approve your valuable pull request.

**Comment by miladsade96** [comment]
And remove all functions and tests related to recursive approach. Thanks ;)

**Pull request closed without merging**